apollo-client/src/main/java/com/ctrip/framework/apollo/internals/InterestedConfigChangeEvent.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.internals;
+
+import com.ctrip.framework.apollo.Config;
+import com.ctrip.framework.apollo.ConfigChangeListener;
+import com.ctrip.framework.apollo.model.ConfigChange;
+import com.ctrip.framework.apollo.model.ConfigChangeEvent;
+import com.ctrip.framework.apollo.spring.annotation.ApolloConfigChangeListener;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * In {@link ApolloConfigChangeListener} you set some interested key's rule, you can get those keys
+ * by this class's instance.
+ *
+ * @author wxq
+ */
+class InterestedConfigChangeEvent extends ConfigChangeEvent {
```

Comment:
Make InterestedConfigChangeEvent  only can be used in same package.

So if people use sdk `apollo-client`, they cannot use InterestedConfigChangeEvent.

Then we don't need to care about the compability if delete `InterestedConfigChangeEvent` in the future.

Reply:
sounds a good idea.